gc/base/MemoryPoolLargeObjects.cpp

```diff
@@ -1013,7 +1016,7 @@ MM_MemoryPoolLargeObjects::redistributeFreeMemory(MM_EnvironmentBase* env, uintp
 	void* oldLOABase = _currentLOABase;
 
 	/* Calculate new LOA size based on current LOA ratio */
-	_loaSize = MM_Math::roundToFloor(_extensions->heapAlignment, (uintptr_t)((float)newOldAreaSize * _currentLOARatio));
+	_loaSize = MM_Math::roundToCeiling(_extensions->heapAlignment, (uintptr_t)((float)newOldAreaSize * _currentLOARatio));
 
 	assume0(_loaSize > 0);
 
```

Comment:
Since the change is mostly about heap alignmented effects, could/should we active this assertion (and similar) a few lines lower?

`assume0(MM_Math::roundToFloor(_extensions->heapAlignment, _soaSize) == _soaSize);`

Reply:
done

Reply:
there is another, similar one, early in expandWithRange

Reply:
done